sregistry/client/__init__.py

```diff
@@ -71,24 +71,24 @@ def get_parser():
     inspect = subparsers.add_parser("inspect",
                                     help="inspect an image in your database")
 
-    inspect.add_argument("query", nargs=1,
+    inspect.add_argument("query",
                           help="image search query to inspect", 
                           type=str, default=None)
 
     # Get path to an image
     get = subparsers.add_parser("get",
                                 help="get an image path from your storage")
 
-    get.add_argument("query", nargs='*', 
+    get.add_argument("query",
                      help="image search query to inspect", 
-                     type=str, default="*")
+                     type=str, default=None)
 
```

Comment:
If you add a default as None, then that would need to be checked for. If you don't add a default, would it error without it (as it's a required positional argument?) For some of these I see you added None, and others not, could you explain this decision?

Reply:
Actually if you give it no arguments, you already get an appropriate error message:

```
$ sregistry get
usage: sregistry get [-h] query
sregistry get: error: the following arguments are required: query
```

Reply:
This is with the default as None - then is the default needed at all?

Reply:
It's probably pointless indeed. Will remove it.